lib/ruby_terraform/commands/apply.rb

```diff
@@ -1,48 +1,35 @@
-require 'json'
 require_relative 'base'
+require_relative '../command_line/options'
 
 module RubyTerraform
   module Commands
     class Apply < Base
-      def configure_command(builder, opts)
-        directory = opts[:directory]
-        plan = opts[:plan]
-        vars = opts[:vars] || {}
-        var_file = opts[:var_file]
-        var_files = opts[:var_files] || []
-        target = opts[:target]
-        targets = opts[:targets] || []
-        state = opts[:state]
-        input = opts[:input]
-        auto_approve = opts[:auto_approve]
-        no_backup = opts[:no_backup]
-        backup = no_backup ? '-' : opts[:backup]
-        no_color = opts[:no_color]
+      def command_line_options(option_values)
+        RubyTerraform::CommandLine::Options.new(
+          option_values: option_values,
+          command_arguments: {
```

Comment:
I see now that this is actually the definitions of options. I wonder if the parameter names should be `definitions` and `values`. Again the `option_` is superfluous in my view since it is being passed to a class called `Options`.

Reply:
I was really struggling with the naming.. Trying to differentiate between the parameters supplied by the user in their hash, and the parameters we're passing to terraform. I'll have another crack and see if I can think of something better.